packages/opencensus-core/src/stats/view.ts

```diff
@@ -187,9 +196,58 @@ export class BaseView implements View {
     }
   }
 
+  /**
+   * Gets view`s metric
+   * @returns {Metric}
+   */
+  getMetric(): Metric {
+    const {type} = this.metricDescriptor;
+    let startTimestamp: Timestamp;
+
+    switch (type) {
+      case MetricDescriptorType.GAUGE_INT64:
+      case MetricDescriptorType.GAUGE_DOUBLE:
+        startTimestamp = null;
+        break;
+      default:
+        const [seconds, nanos] = process.hrtime();
+        startTimestamp = {seconds, nanos};
+    }
+
+    const timeseries: TimeSeries[] = [];
+
+    Object.keys(this.rows).forEach(key => {
+      const {tags} = this.rows[key];
+      const labelValues: LabelValue[] = MetricUtils.tagsToLabelValues(tags);
+      const point: Point = this.toPoint(startTimestamp, this.getSnapshot(tags));
+      timeseries.push({startTimestamp, labelValues, points: [point]});
+    });
+
+    return {descriptor: this.metricDescriptor, timeseries};
+  }
+
+  /**
+   * Converts snapshot to point
+   * @param timestamp The timestamp
+   * @param data The aggregated data
+   * @returns {Point}
+   */
+  private toPoint(timestamp: Timestamp, data: AggregationData): Point {
+    let value;
+    if (data.type === AggregationType.DISTRIBUTION) {
+      // TODO: Add examplar transition
+      value = Object.assign(
+          {bucketOptions: {explicit: {bounds: data.buckets}}}, data);
+    } else {
+      value = data as SummaryValue;
```

Comment:
We don;t generate SummaryValue inside stats, please remove this.
For referenc,  please check all toPoint methods here, https://github.com/census-instrumentation/opencensus-java/blob/master/impl_core/src/main/java/io/opencensus/implcore/stats/MutableAggregation.java

Reply:
Could you please clarify this. 
According to proto description, the point has [value property](https://github.com/census-instrumentation/opencensus-node/blob/cb79b9cf5529dcb7a5b139539ee1a01d066eb056/packages/opencensus-core/src/metrics/export/types.ts#L157) ([proto link](https://github.com/census-instrumentation/opencensus-proto/blob/b11a67434194733b34c11f206938263fa16ad1cf/src/opencensus/proto/metrics/v1/metrics.proto#L175))
Which can has a `SummaryValue`

Reply:
> According to proto description, the point has value property (proto link)

Yes you're right. But stats API doesn't produce summaryValue. 

Below conversion are expected here:
AggregationType.DISTRIBUTION (or DistributionData) -> Point ```DistributionValue```.
All other Aggregation (SumData, CountData and LastValueData) -> Point ```number```.

@bogdandrutu Correct me if I am wrong.

Reply:
Correct our stats package should not produce Summary.

Reply:
Done